awscli/examples/healthlake/create-fhir-datastore.rst

```diff
@@ -17,4 +17,31 @@ Output::
         "DatastoreId": "(Datastore ID)"
     }
 
-For more information, see `Creating and monitoring a FHIR Data Store <https://docs.aws.amazon.com/healthlake/latest/devguide/working-with-FHIR-healthlake.html>`__ in the *Amazon HealthLake Developer Guide*.
+**Example 2: Create a SMART on FHIR-enabled HealthLake data store**
+

```

Comment:
Missing the "The following ``create-fhir-datastore`` example" introductory line.

Reply:
There is a "The following ``create-fhir-datastore`` example" introductory line at the top. Should I add it for "**Example 2:" somewhere?

Reply:
Added it.